ui/src/shared/components/CustomTimeRange.js

```diff
@@ -82,6 +124,41 @@ class CustomTimeRange extends Component {
       onClose()
     }
   }
+
+  handleMovingTimeRange(movingTimeRange) {
```

Comment:
this should not be called `movingTimeRange` if this is not about moving time ranges